lib/rubocop/cop/style/def_with_parentheses.rb

```diff
@@ -11,35 +11,41 @@ module Style
       #
       #   # bad
       #   def foo()
-      #     # does a thing
+      #     do_something
       #   end
       #
       #   # good
       #   def foo
-      #     # does a thing
+      #     do_something
       #   end
       #
-      #   # also good
-      #   def foo() does_a_thing end
+      #   # bad
+      #   def foo() = do_something
+      #
+      #   # good
+      #   def foo = do_something
+      #
+      #   # good
```

Comment:
Should we add an explanation here? E.g., `# good (without parentheses it's a syntax error)`.

Reply:
Good point! I've updated it!